bucket/coreftp.json

```
@@ -0,0 +1,44 @@
+{
+    "version": "2.2",
+    "homepage": "http://www.coreftp.com",
+    "description": "A FTP client",
+    "license": {
+        "identifier": "freeware",
+        "url": "http://www.coreftp.com/license.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "http://www.coreftp.com/download/coreftplite64.exe#/dl.7z",
+            "hash": "740299de93adaca8c17abd1425ec7ff49b14a857cc1e3879d6005da7c10e60c7"
+        },
+        "32bit": {
+            "url": "http://www.coreftp.com/download/coreftplite.exe#/dl.7z",
+            "hash": "20db16ef37a5dc26016802172518da1e50a8e7a90cf75828fccf9474df2fffde"
+        }
+    },
+    "bin": [
+        [
+            "coreftpu.exe",
+            "coreftp.exe",
+            "corecmd.exe"
+        ]
+    ],
```

Any of this executable is working from command line. Post screenshot of any command execution with parameters and result.

```suggestion
    "pre_install": "Remove-Item \"$dir\\uninst*\" -Force",
    "bin": [
        "coreftpu.exe",
        "coreftp.exe",
        "corecmd.exe"
    ],
```

---

I have deleted `coreftp.exe` and `corecmd.exe`, and renamed `coreftpu.exe` to `coreftp.exe`.